packages/maskbook/src/extension/options-page/DashboardDialogs/Persona/Create.tsx

```diff
@@ -45,7 +52,7 @@ export function DashboardPersonaCreateDialog(props: WrappedDialogProps) {
                                 required
                                 label={t('name')}
                                 value={name}
-                                onChange={(e) => setName(e.target.value)}
+                                onChange={onChange}
```

Comment:
suggestion:

Add a [`maxlength`][1] to the `input` element

[1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength